### PR TITLE
i18n(es): Updated `frontmatter.md`, `components.mdx` & `configuration.md`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `frontmatter.md`, `components.mdx` & `configuration.md`
> In `configuration.md` there were some sections missing which do not appear in any commit but I have added them to the update

`frontmatter.md`
* [`048e948`](https://github.com/withastro/starlight/commit/048e948bce650d559517850c73d827733b8164c4)

`components.mdx`
* [`faa70de`](https://github.com/withastro/starlight/commit/faa70de584bf596fdd7184c4a8622d67d1410ecf)

`configuration.md`
* [`8102389`](https://github.com/withastro/starlight/commit/810238934ae1a95c53042ca2875bb4033aad0114#diff-258b8e475fefcd9503466ed259ff2913e1dd52a90fd46170aca60318c67459ce)
* [`048e948`](https://github.com/withastro/starlight/commit/048e948bce650d559517850c73d827733b8164c4#diff-258b8e475fefcd9503466ed259ff2913e1dd52a90fd46170aca60318c67459ce)

### CODE REVIEW

File: docs/src/content/docs/es/reference/configuration.md [hunk 1]
```diff
@@ -288,3 +349,12 @@ interface HeadConfig {
...
 ```
+
+### `lastUpdated`
+
+**type:** `boolean`  
+**default:** `false`
+
+Controla si se muestra el pie de página que indica cuándo se actualizó por última vez la página.
+
+De forma predeterminada, esta función se basa en el historial Git de tu repositorio y puede no ser precisa en algunas plataformas de implementación que realizan [clonaciones superficiales](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una página puede anular esta configuración o la fecha basada en Git utilizando el campo [`lastUpdated`](/reference/frontmatter/#lastupdated) en el frontmatter.
```
Comment (dreyfus92) [inline docs/src/content/docs/es/reference/configuration.md:360]:
```suggestion
De forma predeterminada, esta función se basa en el historial Git de tu repositorio y puede no ser precisa en algunas plataformas de implementación que realizan [copias superficiales](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una página puede anular esta configuración o la fecha basada en Git utilizando el campo [`lastUpdated`](/reference/frontmatter/#lastupdated) en el frontmatter.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo una sugerencia @Waxer59, buen trabajo 😁

File: docs/src/content/docs/es/guides/components.mdx [hunk 2]
```diff
@@ -101,3 +101,28 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
 ```
 
 :::
+
+### Iconos
+
+import { Icon } from '@astrojs/starlight/components';
+import IconsList from '../../../components/icons-list.astro';
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:108]: Thanks @Waxer59!

I think we need another level of `../` here to import correctly:

```suggestion
import IconsList from '../../../../components/icons-list.astro';
```

Comment (dreyfus92) [reply docs/src/content/docs/es/guides/components.mdx:108]: Nice catch Chris 🚀

**Comment by Waxer59** [comment]
Thanks for the review @delucis & @dreyfus92 🤠

**Review by delucis — APPROVED** [review]
Thank you @Waxer59 & @dreyfus92! Looks good now — so will merge 🙌

**Pull request merged**